### DISCUSSION TITLE
Should I run these updates?

### DISCUSSION THREAD

**Discussion by SonOvaSailor** [Guilouz/Klipper-Flsun-Speeder-Pad#184] [Q&A]
This klipper install is running great! Thanks Guilouz! So my Mainsail interface is saying I need to run several updates. My question is should I? Have you guys been updating with no problem? 

Any help is greatly appreciated! :)

See Pic. 
![v400 klipper updates needed](https://github.com/user-attachments/assets/9009b727-1270-4dd9-bc30-1575091e353a)

**Comment by lin-ycv** [comment]
if there's nothing wrong witht the printer, best not to update.

**Reply by SonOvaSailor** [reply]
Works great, one question tho, when my Z axis is too high and I adjust it via the speeder pad (in the last case i had to go down .50mm) there is no option to save to Config file as with my other Klipper machines. I've run the "SAVE" macro but there is no change. Do you know how to save this? See Pic.
![20240802_234327](https://github.com/user-attachments/assets/540d6110-8b8d-4ec6-bcdf-b8e69af10b17)

**Reply by lin-ycv** [reply]
I believe if you adjust the z height while printing, it's saved automatically.

**Reply by SonOvaSailor** [reply]
...it stays on -0.500 on the display and this setting in the Config file never changes. I may be wrong about this and am open to being corrected :), in fact I hope i'm wrong and you know what I need to to have the auto-save for the Z offset work for me :), either way thanks for the prompt reply!
![image](https://github.com/user-attachments/assets/63657755-e7b5-4a8f-8254-3736fb432c3e)

**Reply by lin-ycv** [reply]
The offset in your photo is the distant from probe point to nozzle, the -0.5 is a applied on top of the first.
